packages/cli-repl/src/smoke-tests.ts

```diff
@@ -420,13 +420,14 @@ async function runSmokeTest({
     input,
     output,
     stdout,
-    stderr,
+    stderr: includeStderr ? stderr : '',
     executable,
     actualExitCode,
     args: args.map((arg) => redactURICredentials(arg)),
   };
   try {
     assert.match(includeStderr ? `${stdout}\n${stderr}` : stdout, output);
+    assert.doesNotMatch(stderr, /ExperimentalWarning/);
```

Comment:
I'm tempted to change this to match more broadly just `Warning`, but I'm not sure if that might result in false positives.